app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -3189,6 +3205,26 @@ class ChatActivity :
                                 Integer.parseInt(it)
                             }
 
+                            try {
+                                val mostRecentCallSystemMessage = adapter?.items?.first {
+                                    it.item is ChatMessage &&
+                                        (it.item as ChatMessage).systemMessageType != null &&
+                                        (it.item as ChatMessage).systemMessageType!!.ordinal <=
+                                        ChatMessage.SystemMessageType.CALL_TRIED.ordinal &&
+                                        (it.item as ChatMessage).systemMessageType!!.ordinal >=
+                                        ChatMessage.SystemMessageType.CALL_STARTED.ordinal
```

Comment:
i don't like this solution with ordinal too much.
if anything changes in the enum order it could introduce strange bugs someday.
better would be to check for every single CALL_ item?

Reply:
Yeah, you're right. It would be better to have it not rely on any arbitrary placement of ordinals.